pom.xml

```diff
@@ -26,13 +23,32 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
+    <developers>
+        <developer>
+            <!-- Original project author -->
+            <id>cbornet</id>
+        </developer>
+    </developers>
+    <issueManagement>
+        <system>Github</system>
+        <url>https://github.com/OpenAPITools/jackson-databind-nullable/issues</url>
+    </issueManagement>
+    <organization>
+        <name>OpenAPI Tools</name>
+        <url>https://openapitools.org/</url>
+    </organization>
 
     <properties>
-        <!-- Generate PackageVersion.java into this directory. -->
-        <packageVersion.dir>org/openapitools/jackson/nullable</packageVersion.dir>
-        <packageVersion.package>${project.groupId}.jackson.nullable</packageVersion.package>
         <java.version>8</java.version>
 
+        <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ssZ</maven.build.timestamp.format>
+        <!-- TODO: Update on build. Used for reproducible builds. -->
+        <project.build.outputTimestamp>2025-10-30T23:03:53Z</project.build.outputTimestamp>
```

Comment:
Currently the project just inherits the value from the parent pom, but this value should be updated when builds happen.

So what I have here matches existing behavior, but the existing behavior isn't really right.